docs/governance/community-groups.md

```diff
@@ -22,6 +22,46 @@ Non-technical issues or high-level design decisions may be escalated to the [SC]
 
 Technical issues should be resolved by the relevant community group. Failure to resolve technical issues may be escalated to the [SC] for guidance on finding a resolution.
 
+## Working Groups
+
+The CCC project is split into several separate, but related, working groups. Each working group focuses on a specific sub-area of the CCC problem space.
+
+### Working Groups Structure
+
+Working groups are structured with distinct roles to ensure effective operation and governance
+
+1. Working Group Lead: The lead orchestrates the group’s activities, liaises with the broader project community, and represents the group in larger project discussions. As the primary contact, the lead ensures smooth communication and progress.
+2. Maintainers: These are project maintainers deeply invested in the group’s focus areas. Maintainers will have the responsibility of merging pull requests (PRs). While a maintainer may also serve as the lead, it is not a requirement.
+3. PR Approvers: Selected members are entrusted with the authority to review, approve pull requests (PRs), maintaining the quality and integrity of contributions.
+4. Group Members: Contributors who actively engage in the group’s work, from submitting work to reviewing PRs.
+
+The working group charter must encapsulate the members’ details and their respective roles. Newcomers desiring to join should propose their inclusion via a PR to the group members list.
+
+Regular audits of participation are vital. Inactive members should be pruned from the members list to reflect current engagement levels accurately, facilitating more strategic planning of group tasks.
```

Comment:
Is this manual? Or automatic?

Reply:
In my head that would be a manual task, unless there was a way to automate it.

Reply:
Should we add this as a responsibility of the maintainers. I also think there should be some list somewhere saying these people are the active contributors of the WG

Reply:
Probably shared between working group lead and the maintainers? 

Is the list of active contributers that you are suggesting different to the list covered in the working groups structure section?

Reply:
In one of the previous sessions, Eddie and Adrian (iirc) were talking about what constitutes an active member and how you got to be one. Might be worth flagging in the text that there is some discussion required elsewhere about the nature of an active contributor which still needs to be defined.

Reply:
I have included a note to the appropriate section to address this.